libs/openFrameworks/gl/ofShader.cpp

```diff
@@ -446,18 +447,10 @@ string ofShader::parseForIncludes( const string& source, vector<string>& include
 		// while skipping whitespace, read in tokens for: pragma, include, and filename
 		s >> std::skipws >> p >> i >> f;
 
-		if (p.empty())
-			return false;
-		else if (p == "include") {
-			f = i;
-			i = p;
-		}
-		else if (p != "pragma")
-			return false;
-
-		if (i.empty() || (f.size() < 2) ) 
-			return false;
+		if (p.empty() || i.empty() || (f.size() < 2) ) return false;
+		// -----| invariant: all tokens have values
 
+		if (p != "pragma") return false;
```

Comment:
Just want to check, these changes don't undo @patriciogonzalezvivo's  lygia fixes?

Reply:
ah, that would be good to double check. I thought I used the nightly after his changes were merged.